gapic/src/main/com/google/gapid/views/TracerDialog.java

```diff
@@ -458,6 +461,41 @@ protected void configureDialog(DirectoryDialog dialog) {
             friendlyName = "";
           }
         });
+        createLabel(this, "");
+        advanced = withLayoutData(
+            createCheckbox(this, "Advanced", models.settings.traceAdvanced),
+            new GridData(SWT.FILL, SWT.FILL, true, false));
+
+
+        observeFramebuffers = null;
+        ofl = null;
+        if (models.settings.traceAdvanced) {
+          ofl = createLabel(this, "");
+          observeFramebuffers = withLayoutData(
+              createCheckbox(this, "Observe Framebuffers", models.settings.traceObserveFramebuffers),
+              new GridData(SWT.FILL, SWT.FILL, true, false));
+        }
+
+        Listener advancedListener = e -> {
```

Comment:
This is definitely not what we want. But I dont know swt enough to make anything cleaner.

Reply:
The SWT widgets' method .setVisible(boolean) might be helpful. You may use it to replace creating and disposing.

Reply:
It's worth noting that after .setVisible(false), the widget will still occupy a space, it just goes transparent. So if there's other setting checkboxes following in the future, there would be a weird blank line in the middle. I think maybe you can:
1.Make the dependent checkbox staying on the same line, and use .setVisible(false) when unwanted. No blank line.
2.Give the dependent checkbox some indent, and use .setEnabled(false) when unwanted. The checkbox will go grey and the UX is still straightforward.